datasets/dataset.py

```
@@ -196,8 +213,18 @@ def _get_start_indices(self, record: VideoRecord) -> "np.ndarray[int]":
             List of indices of where the frames of each
             segment are to be loaded from.
         """
+        # choose start indices that are spread each time unit given by user
+        if self.time is not None:
+            if record.fps is None:
+                sys.exit("Number of rames per second not provided")
```

Can we raise an exception here instead of sys.exit() ?